frontend/src/Command/Commands/resume.ts

```
@@ -10,10 +14,7 @@ class ResumeCommand extends BaseCommand {
     super("resume", "Resume", "resume", 0, "resume", {});
   }
 
-  getSuggestions(
-    _input: string,
-    _parameters: Record<string, string>
-  ): Promise<SuggestionList> {
+  async getSuggestions({}: SuggestionsParams): Promise<SuggestionList> {
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)